app/helpers/admin/document_collection_group_memberships_helper.rb

```diff
@@ -1,17 +1,23 @@
 module Admin::DocumentCollectionGroupMembershipsHelper
   def document_collection_group_member_title(membership)
-    if membership.non_whitehall_link
-      membership.non_whitehall_link.title
-    else
-      membership.document.latest_edition.title
-    end
+    return membership.non_whitehall_link.title if membership.non_whitehall_link
+
+    membership.document.latest_edition.title
```

Comment:
I do wonder if we're not missing a trick here.

It seems like we now need to perform logic every time we refer to the document title, plus we have the "Unavailable Document" string hardcoded in (at least?) three separate areas of the code.

For example:

1. https://github.com/alphagov/whitehall/blob/5eca54cc92bb3e39226b963b70e2bee0b29d4256/app/views/admin/document_collection_group_memberships/confirm_destroy.html.erb#L8

2. https://github.com/alphagov/whitehall/blob/5eca54cc92bb3e39226b963b70e2bee0b29d4256/app/views/admin/document_collection_group_memberships/index.html.erb#L59-L60

3. https://github.com/alphagov/whitehall/blob/5eca54cc92bb3e39226b963b70e2bee0b29d4256/app/views/admin/document_collection_group_memberships/reorder.html.erb#L14

This feels like something that could be relocated to the helper, so the view has to do less work and is less repetitive.

What about something like this to DRY it up a little?

```suggestion
    return tag.span("Unavailable Document", class: "govuk-!-font-weight-bold") if document_collection_group_member_unavailable?(membership)

    membership.document.latest_edition.title
```

Reply:
As mentioned above, i attempted to put it as part of the title but in that sense the helper hid away the entire logic which also felt convoluted to me. Also because there's already that logic in the view, i.e. don't show the "view" link, in addition to the title change. 

After this i wanted to refactor the views to introduce a partial which might make this a bit easier to read, so might reintroduce the title load there.